media/linux/pds-sqlite3-queries/training_sheets.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+# Class information for the different sheets of create-training-rosters.py
+
+import sys
+sys.path.insert(0, '../../../python')
+
+import os
+
+import logging.handlers
```

Comment:
Do you need all of the imports listed below?  Trim out the ones that you're not using in this file.